gtfs/api/routes.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework import serializers, viewsets, filters
+from gtfs.models import Route, Stop
+from gtfs.api.stops import StopSerializer
+
+
+class RouteSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Route
+        fields = ("id", "name", "stops")
+
+    name = serializers.SerializerMethodField()
+    stops = serializers.SerializerMethodField()
+
+    def get_name(self, obj):
+        return obj.short_name
+
+    def get_stops(self, obj):
+        if "stops" not in self.context:
+            self.context["stops"] = Stop.objects.all()
+        stops = self.context["stops"].filter(feed=obj.feed)
+        return StopSerializer(stops, many=True).data
```

Comment:
This functions a bit wrong, it returns all the stops of the feed for every route.

Reply:
This returns what we want, but no idea/time to think if it's suitable/best approach for this case: `Stop.objects.filter(stop_times__trip__route_id=obj.id).distinct()`